MultiVolumeImporterPlugin.py

```diff
@@ -165,12 +177,15 @@ def examineFilesMultiseries(self,files):
     parsing multivolumes out.
     """
 
-    logging.debug('MultiVolumeImportPlugin:examineMultiseries')
+    if self.detailedLogging:
+      logging.debug('MultiVolumeImportPlugin:examineMultiseries')
```

Comment:
Since we are updating the logging, we should be consistent `MultiVolumeImportPlugin` vs `DICOMMultiVolumePlugin `

Reply:
`MultiVolumeImportPlugin` -> `MultiVolumeImporterPlugin`

Reply:
fixed